samples/widgets/editor/src/runtime/widget.tsx

```diff
@@ -74,34 +71,52 @@ export default class Widget extends React.PureComponent<AllWidgetProps<{}>, Stat
   componentDidUpdate = evt => {
 
     if (this.props.useMapWidgetIds && this.props.useMapWidgetIds.length === 0) {
-         if (this.state.currentWidget) {
-            this.state.currentWidget.destroy();
+      if (this.state.currentWidget) {
+        this.state.currentWidget.destroy();
       }
-     }
+    }
   };
 
   render() {
+
+    let mvc = <p>Please select a map.</p>;
+
+    const css = `
+    .esri-editor__scroller {
+      overflow-y: auto;
+      padding-top: $cap-spacing--half;
+      padding-bottom: $cap-spacing;
+      max-height: 1em;
+      }
+      .esri-editor__content-group {
+        max-height: 1em;
+      }
    
-      let mvc = <p>Please select a map.</p>;
+      `
     if (
       this.props.hasOwnProperty("useMapWidgetIds") &&
       this.props.useMapWidgetIds &&
-      this.props.useMapWidgetIds.length === 1
+      this.props.useMapWidgetIds[0]
+      
     ) {
       mvc = (
         <JimuMapViewComponent
-          useMapWidgetIds={this.props.useMapWidgetIds}
+          useMapWidgetId={this.props.useMapWidgetIds?.[0]}
```

Comment:
I did not know this property (`useMapWidgetId`) was available - is this a better way to do it than to use `useMapWidgetIds`? I think probably so ... if so, should we update the rest of the examples to follow this pattern?

Reply:
Update: this is a change in v1.4, so we will be updating the samples.